prog/vm/github_runner.rb

```diff
@@ -184,6 +184,16 @@ def before_run
       # ./env.sh sets some variables for runner to run properly
       ./actions-runner/env.sh
 
+      # The runner script loads environment variables from .env file but ignores
+      # the standard /etc/environment file of the operating system.
```

Comment:
```suggestion
      # Include /etc/environment in the runner environment, it's
      # otherwise ignored, and this omission has caused problems.
      #
      # See https://github.com/actions/runner/issues/1703
```

Not my best prose, the main problem I was trying to fix was that the comment status quo ante describes a situation, but doesn't make a normative claim about whether it's bad (we have decided it is bad). With six months distance, the comment and the code can be written as being in contradiction.